lib/ecto_migration_default.ex

```diff
@@ -48,6 +48,12 @@ defimpl EctoMigrationDefault, for: BitString do
   end
 end
 
+defimpl EctoMigrationDefault, for: List do
+  def to_default(value) do
+    inspect(value)
```

Comment:
I'm not sure that this is sufficient. What if it was a list of things that were not supported? i.e `default: [%{a: 1}]`. I think we'd need to do something like:
```elixir
"[" <> Enum.map_join(value, ", ", &EctoMigrationDefault.to_default/1) <> "]"
```

Reply:
Good point! Will add some handling for those cases